ref: convert documentation from README.rst to GitHub Wiki (#495)
linode/linode-cli: README.md

# linode-cli (lin)

The Linode Command Line Interface

> Auto-generated from the [Linode OpenAPI](https://www.linode.com/docs/api/) spec using the [OpenAPI3 Parser](https://github.com/Dorthu/openapi3). Allows easy access to any of the Linode API endpoints and displays the results in a organized, configurable, table.

Reviewer: ```suggestion
Provides easy access to any of the Linode API endpoints from the command line and displays results in an organized, configurable table. 

This project is automatically generated from the [Linode OpenAPI spec](https://www.linode.com/docs/api/) using the [openapi3 Python package](https://github.com/Dorthu/openapi3).
```

![Example of CLI in use](https://raw.githubusercontent.com/linode/linode-cli/main/demo.gif)

Visit the [Wiki](../../wiki) for more information.

## Install

> Install via pypi

Reviewer: ```suggestion
Install via PyPI:
```

```bash
pip3 install linode-cli
```

Visit the [Wiki](../../wiki/Installation) for more information.

## Contributing

> This CLI is generated based on the OpenAPI specification for Linode's API.  As
> such, many changes are made directly to the spec.

Reviewer: ```suggestion
This CLI is generated from the [OpenAPI specification for Linode's API](https://github.com/linode/linode-api-docs).  As
such, many changes are made directly to the spec.
```
